packages/compass-logging/src/index.ts

```diff
@@ -57,10 +57,7 @@ export function createLoggerAndTelemetry(component: string): {
     event: string,
     properties: TrackProps = {}
   ): Promise<void> => {
-    const isTrackingEnabled = (global as any)?.hadronApp?.isFeatureEnabled(
-      'trackUsageStatistics'
-    );
-    if (!isTrackingEnabled) {
+    if (!trackUsageStatistics) {
```

Comment:
Why are we changing the structure of the code here? I really liked the idea of `getPreferenceValue()` in the technical design to allow specifically accessing one property value, which could replace `isFeatureEnabled()` here one-to-one.

I’d rather not add change listeners to every place in which we consume preferences, i.e. avoid tracking the same state variables in multiple locations.

Reply:
Updated.